apps/ccf-rui/src/app/modules/registration-modal/registration-modal/registration-modal.component.ts

```diff
@@ -64,11 +72,47 @@ export class RegistrationModalComponent implements OnInit {
   }
 
   /**
-   * Opens dialog
+   * Opens dialog and opens screen size notice panel if necessary
    */
   openDialog(): void {
-    this.dialog.open(RegistrationContentComponent, {
-      autoFocus: false,
-    });
+    this.dialog
+      .open(RegistrationContentComponent, {
+        autoFocus: false,
+        panelClass: 'registration-modal',
+        hasBackdrop: false,
+        width: '100vw',
+        maxWidth: '100vw',
+        height: '100vh',
+        maxHeight: '100vh',
+      })
+      .afterOpened()
+      .subscribe(() => {
+        const initialStorageValue = booleanAttribute(localStorage.getItem(DEFAULT_SCREEN_SIZE_NOTICE_STORAGE_KEY));
```

Comment:
The screen size notice logic should go in the app component. Not as a nested dialog of the registration model